arbiter-core/src/environment/mod.rs

```diff
@@ -556,6 +562,18 @@ impl Environment {
                                     )),
                                 }
                             }
+                            EnvironmentData::TransactionCount(address) => {
+                                let db = evm.db().unwrap();
+                                let recast_address = revm::primitives::Address::from(address);
+                                match db.accounts.get(&recast_address) {
+                                    Some(account) => {
+                                        Ok(Outcome::QueryReturn(account.info.nonce.to_string()))
```

Comment:
You could just set up its own return type so you don't have to stringify

Reply:
Why would we have a unique type for this one but not any of the others?

Reply:
I'm confused then. Why did you not use the `EnvironmentData` enum and the `Instruction::Query`? It seems weird to use a mix of the two. That's why I don't understand this.

Reply:
I don't understand what you mean, every quarry does that. It's the exact same convention of all the other quarries no?